cf/i18n/resources/de-de.all.json

```diff
@@ -2909,15 +2909,15 @@
   },
   {
     "id": "Maximum amount of memory an application instance can have (e.g. 1024M, 1G, 10G)",
-    "translation": "Maximalwert für den möglichen Speicher einer Anwendungsinstanz (z.B. 1024M, 1G, 10G)"
+    "translation": "Maximalwert für den möglichen Speicher einer Anwendungsinstanz (z. B. 1024M, 1G, 10G)"
```

Comment:
The German person on my team (and some googling I did) seem to indicate it's more common to not use a space in `z.B.`. Why this change?